website/docs/stl.md

```diff
@@ -1,18 +1,20 @@
-stl - This page was auto-generated. Feel free to help us improve the documentation by creating a pull request.
+Specifies which C++ Standard Library to use.
 
 ```lua
-stl (value)
+stl ("value")
 ```
 
+The `staticruntime` API is used to determine if a static or shared version of the STL is used.
+
 ### Parameters ###
 
 `value` is one of:
 
-* `none`: needs documentation
-* `gabi++`: needs documentation
-* `stlport`: needs documentation
-* `gnu`: needs documentation
-* `libc++`: needs documentation
+* `none`: Minimal C++ runtime library.
+* `gabi++`: C++ runtime library
+* `stlport`: STLport runtime library
+* `gnu`: GNU STL library
+* `libc++`: LLVM libc++ library
```

Comment:
Sorry for extremely minor comment but... to be consistent, periods everywhere or nowhere  😄

Reply:
Haha, I must have been a bit distracted doing this one, good catch 😄